sdk/spring-sdk/README.md

```diff
@@ -0,0 +1,232 @@
+# Kalix Spring SDK
+
+(Note: since the Spring SDK is very recent, there is not yet proper documentation so this document aims to serve as a general introduction guide.)
+
+The goal of the Spring SDK is to provide a code-first approach to developing Kalix applications so that a developer does not need to worry about learning protobuf upfront.
+
+This SDK offers a way to develop applications which should be familiar to someone who has worked with Spring applications by making use of annotations to assemble Kalix and its components.
+
+If you're new to Kalix and the different types of entities that exist in a Kalix app, consider reading [this section](https://docs.kalix.io/services/programming-model.html#_what_is_a_kalix_service) before continuing.  
+
+## Features
+
+As the Spring SDK is more recent than their gRPC-first counterparts, not all features are supported at this time. However, there's already a lot to play with as described next.
+
+### Value Entities
```

Comment:
I think the first step should be to show how to start a new project using the new archetype.